install.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# check Ubuntu version
+source /etc/os-release
+
+if [[ $UBUNTU_CODENAME != 'jammy' ]]
+then
+    echo "Ubuntu 22.04.1 LTS (Jammy Jellyfish) is required"
+    echo "You are using $VERSION"
+    exit 1
+fi
+
+### Get directory where this script is installed
+BASEDIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+### Install ROS2
+cd ~
+git clone https://github.com/Tiryoh/ros2_setup_scripts_ubuntu.git
+./ros2_setup_scripts_ubuntu/run.sh
+
+source /opt/ros/humble/setup.bash
+mkdir -p ~/mini_pupper_ws/src
+cd ~/mini_pupper_ws/src
```

Comment:
How about using `~/ros2_ws/src`  to match the official tutorial?

https://docs.ros.org/en/humble/Tutorials/Beginner-Client-Libraries/Creating-A-Workspace/Creating-A-Workspace.html#create-a-new-directory

Reply:
OK, will do